catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java

```diff
@@ -43,6 +48,8 @@
 public class HistorianPlugin implements PostIngestPlugin, PreIngestPlugin {
     private static final Logger LOGGER = LoggerFactory.getLogger(HistorianPlugin.class);
 
+    private static final String HISTORY_METACARDS = "history-metacards";
```

Comment:
We could.. but this is a bundle. Should I move it to the HistoryMetacardImpl instead maybe?